force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -218,14 +249,12 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static List<Integer> createIntegerList(Integer n) {
         // Create a new list of integers 
-
-        /*
-        for() {
+        List<Integer> listOfNumbers = new List<Integer>();
+        for(Integer i = 0; i < n; i++) {
             //add integer to the list
-        }
-        */ 
-
-        return null; // Replace null with the variable you used to store the result
+            listOfNumbers.add(i + 1);
+        } 
+        return listOfNumbers; // Replace null with the variable you used to store the result
```

Comment:
Could you just start your index = 1 ?

Challenge try and do it with a while loop

Reply:
Am I able to do that? Would it skip the index of 0 then? Is that first component in the parentheses after the "for" just saying where the loop will start in the list?

Ok, I'll try it with the while loop!

Reply:
In this case you are not using the loop variable to index a list. You are just using to creat a number from  i...n